doc/player.md

```diff
@@ -0,0 +1,12 @@
+# Player API
+
+mimic mtg player_api
+
+## NOTE
+
+`xcompat.player.player_attached`
+
+read/write from it is fine, looping over it is not as it is a proxy table. s
```

Comment:
"**This** would need..."?

Reply:
https://github.com/mt-mods/xcompat/pull/41/commits/a0026f4c8221f9b6d53af5baaee415e511b0ac39